### DISCUSSION TITLE
Test for crud

### DISCUSSION THREAD

**Discussion by methos02** [orchidsoftware/platform#2494] [Q&A] [answered]
Hello, 

First, thx for your job. 

I try to create a test for crud but don't know how do it. It's a very sensitive part of this application and I really need it. 

Thx in advance.

**Comment by methos02** [comment]
@tabuna, no idea ? I read the documentation without success :(

**Reply by tabuna** [reply]
Hello, you can use [HTTP requests to test](https://laravel.com/docs/9.x/http-tests). For CRUD, I did not make any helpers in this regard.

**Comment by methos02** [comment, marked as answer]
Thx for your answer,

Problème solved, i forget the "/save" at the end of the url. (ex: route('platform.systems.model.create') . '/save'))
